### PR TITLE
fix codestyle

### PR DESCRIPTION
**Pull request by SharplEr**

### CODE REVIEW

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java [hunk 1]
```diff
@@ -126,6 +141,7 @@ public final void sendAndReceiveStatBaseline(IoSendReceiveBaselineState state, E
         sendAndReceive(state, counters);
     }
 
+    /** */
```
Comment (NSAmelchev) [inline modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java]: Add description.

Comment (NSAmelchev) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java]: And other classes.

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java [hunk 2]
```diff
@@ -170,7 +187,11 @@ public final void setup() {
         }
     }
 
-    public final void sendAndReceive(IoState state, EventCounters counters) {
+    /**
```
Comment (NSAmelchev) [inline modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java:190]: Add a normal description.

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java [hunk 3]
```diff
@@ -427,29 +447,29 @@ private static Type getType(String comp0, String ssl0) {
...
+            if ("NO_COMPRESSION".equals(comp) && ssl)
                 return Ssl;
-            if (comp.equals("NO_COMPRESSION") && !ssl)
+            if ("NO_COMPRESSION".equals(comp) && !ssl)
                 return No;
-            if (comp.equals("LZ4") && !ssl)
+            if ("LZ4".equals(comp) && !ssl)
                 return LZ4;
-            if (comp.equals("LZ4") && ssl)
+            if ("LZ4".equals(comp) && ssl)
```
Comment (NSAmelchev) [inline modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java]: Maybe to use static final strings for types?

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java [hunk 4]
```diff
@@ -22,129 +22,161 @@
 
 /** */
```
Comment (NSAmelchev) [inline modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java]: Add description.

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java [hunk 5]
```diff
@@ -22,129 +22,161 @@
...
 final class RegularDataUtils {
+    /**
+     * Return regular data with statistics close to native texts.
+     *
+     * @param size Size of generating data.
+     * @param n Number of words.
```
Comment (NSAmelchev) [inline modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java]: Do you sure that is a number of words?

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java [hunk 6]
```diff
@@ -22,129 +22,161 @@
...
+     * @return Regular data.
+     */
+    static byte[] generateRegularData(int size, int n) {
+        ByteBuffer buf = ByteBuffer.allocate(size);
+
+        Language language = generateLanguage(n);
+
+        Random random = new Random(3L);
```
Comment (NSAmelchev) [inline modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java]: Rename to rnd ?

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java (hunk 2 above)
Comment (SharplEr) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java:190]: Done

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java (hunk 1 above)
Comment (SharplEr) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java]: Done.

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java (hunk 3 above)
Comment (SharplEr) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/GridTcpCommunicationSpiBenchmark.java]: Done

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java (hunk 4 above)
Comment (SharplEr) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java]: Done

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java (hunk 5 above)
Comment (SharplEr) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java]: Done

File: modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java (hunk 6 above)
Comment (SharplEr) [reply modules/benchmarks/src/main/java/org/apache/ignite/internal/benchmarks/jmh/tcp/RegularDataUtils.java]: Done

**Pull request merged**